<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlainTextEditor</class>
 <widget class="QWidget" name="PlainTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="textPage"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="loadingLabel">
         <property name="text">
          <string>Loading...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="characterLabel">
         <property name="text">
          <string>Character</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lfLabel">
         <property name="text">
          <string>LF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="encodingLabel">
         <property name="text">
          <string>UTF-8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
